addons/crm/models/crm_team.py

```diff
@@ -138,6 +139,40 @@ def write(self, vals):
                 })
         return result
 
+    def unlink(self):
+        """ When unlinking, concatenate ``crm.lead.scoring.frequency`` linked to
+        the team into "no team" statistics. """
+        frequencies = self.env['crm.lead.scoring.frequency'].search([('team_id', 'in', self.ids)])
+        if frequencies:
+            existing_noteam = self.env['crm.lead.scoring.frequency'].sudo().search([
+                ('team_id', '=', False),
+                ('variable', 'in', frequencies.mapped('variable'))
+            ])
+            for frequency in frequencies:
+                # skip void-like values
+                if float_compare(frequency.won_count, 0.1, 2) != 1 and float_compare(frequency.lost_count, 0.1, 2) != 1:
+                    continue
+
+                match = existing_noteam.filtered(lambda frequ_nt: frequ_nt.variable == frequency.variable and frequ_nt.value == frequency.value)
+                if match:
+                    # avoid adding 0.1 + 0.1, used to avoid 0.0 and have a starting value
```

Comment:
I don't really understand this comment?

Reply:
When initializing frequencies, won and lost count are set to 0.1 and not 0 to avoid side effects in computation. I respect the "0.1 means 0", aka, not adding them (avoid having 0.2, as it should stay at 0.1, and have 0.1 if final result is 0).

Reply:
Then I think a more elaborate comment just like your answer here would be nice.